src/thunderbots/ai/world/robot.h

```diff
@@ -0,0 +1,98 @@
+#ifndef AI_WORLD_ROBOT_H_
+#define AI_WORLD_ROBOT_H_
+
+#include "geom/angle.h"
+#include "geom/point.h"
+#include "thunderbots_msgs/Robot.h"
+
+/**
+ * Defines an SSL robot
+ */
+class Robot
+{
+   public:
+    // The max radius of a robot in metres
+    static constexpr double MAX_RADIUS = 0.09;
+
+    /**
+     * Creates a new robot given a pattern id
+     */
+    explicit Robot(unsigned int id);
+
+    /**
+     * Creates a new robot given a pattern id
+     */
+    explicit Robot(const thunderbots_msgs::Robot& robot_msg);
+
+    /**
+     * Updates the state of the robot.
+     *
+     * @param new_position the new position of the robot
```

Comment:
same as field: units would be nice :)

Reply:
Fixed. Also fixed more weird line wrapping.